cmd/bacalhau/describe.go

```diff
@@ -131,18 +141,24 @@ func describe(cmd *cobra.Command, cmdArgs []string, OD *DescribeOptions) error {
 		jobDesc.Status.LocalEvents = localEvents
 	}
 
-	b, err := model.JSONMarshalWithMax(jobDesc)
+	b, err := model.JSONMarshalIndentWithMax(jobDesc,3)
```

Comment:
Your PR message said you're using 4 spaces but this seems to output 3... did you mean to use 3 or 4?

Reply:
Yes, I reduced it to 3 after. I thought it looked better